client/src/components/Lists.js

```diff
@@ -1,21 +1,38 @@
+import React from 'react';
 import { Card } from 'primereact/card';
-import { useHistory } from 'react-router-dom';
 import '../styles/Lists.css';
-import { database } from '../fakeData/database';
+import { db } from '../fakeData/db';
 
-export const Lists = () => {
-  const history = useHistory();
-  const handleClick = index => {
-    history.push(`/lists/${index}`);
+export class Lists extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { lists: [] };
+  }
+
+  componentDidMount() {
+    this.getLists();
+  }
+
+  getLists = async () => {
+    const lists = await db.getLists();
+    this.setState({ lists: lists });
+  };
+
+  handleClickCard = id => {
```

Comment:
Same as above

Reply:
Also since you will be using arrow function version in the JSX, you can just use regular non arrow function here